exporter/signalfxexporter/exporter.go

```diff
@@ -230,3 +264,24 @@ func (se *signalfxExporter) pushLogs(ctx context.Context, ld plog.Logs) error {
 	_, err := se.pushLogsData(ctx, ld)
 	return err
 }
+
+func buildHeaders(config *Config) map[string]string {
+	headers := map[string]string{
+		"Connection":   "keep-alive",
+		"Content-Type": "application/x-protobuf",
+		"User-Agent":   "OpenTelemetry-Collector SignalFx Exporter/v0.0.1",
```

Comment:
I wonder if there is a means to get the published collector version to help capture analytics within SignalFx?

Reply:
Totally. Actually, I think that could be done via an extension or using the host BuildInfo. Again, felt like a bridge too far for those changes. It is worth filing an issue to follow up imo.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16841 is filed to follow up.